core/os/memory.h

```diff
@@ -158,7 +152,7 @@ T *memnew_arr_template(size_t p_elements, const char *p_descr = "") {
 
 		/* call operator new */
 		for (size_t i = 0; i < p_elements; i++) {
-			new (&elems[i], sizeof(T), p_descr) T;
+			new (&elems[i]) T;
```

Comment:
This makes `p_descr` unused. If that's fine, it should be removed.

Reply:
Fixed!